examples/plot_decoding_tutorial.py

```diff
@@ -47,9 +50,9 @@
 # using :func:`nilearn.plotting.plot_epi`.
 # We will visualize the previously fetched fmri data from Haxby dataset.
 #
-# Because fmri data is 4D (it consists of many 3D EPI images), we cannot 
-# plot it directly using :func:`nilearn.plotting.plot_epi` (which accepts 
-# just 3D input). Here we are using :func:`nilearn.image.mean_img` to 
+# Because fmri data is 4D (it consists of many 3D EPI images), we cannot
+# plot it directly using :func:`nilearn.plotting.plot_epi` (which accepts
```

Comment:
```suggestion
# Because fmri data are 4D (they consist of many 3D EPI images), we cannot
# plot them directly using :func:`nilearn.plotting.plot_epi` (which accepts
```